packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react'
+import { SettingsContext } from '../../context'
+import css from './style.module.scss'
+
+export default function ConnectionStatus () {
+  const { socket } = React.useContext(SettingsContext)
+
+  const [connected, setConnected] = useState(true)
+  const [status, setStatus] = useState('show')
+
+  useEffect(() => {
+    socket.on('connect', () => {
+        setStatus('show')
+        setConnected(true)
+        setTimeout(() => {
+            setStatus('hidden')
+        }, 700)
```

Comment:
Don't forget to use batch
```suggestion
         batch(() => {
            setStatus('show')
            setConnected(true)
            setTimeout(() => {
                setStatus('hidden')
            }, 700)
        })
```